### PR TITLE
readme: update `list libraries`

### PR DESCRIPTION
**Pull request by idrysdale**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -486,11 +486,13 @@ DELETE /organisations/:orgid/librarians/:userid/
 GET /libraries/
 ```
 
-List all libraries and their library items that belong to me.
+List a collection of libraries (and their library items) that is either:
 
-ℹ️ porting notes: replaces previous route `assessment/loaduserlibraries`
+a) a gloabl library
```
Comment (paulfurley) [inline README.md]:
```suggestion
a) a global library
```

File: README.md [hunk 2]
```diff
@@ -486,11 +486,13 @@ DELETE /organisations/:orgid/librarians/:userid/
...
+b) a library that belongs to me,
+b) a library belonging to an organisations I'm a member of
```
Comment (paulfurley) [inline README.md]:
```suggestion
c) a library belonging to an organisation I'm a member of
```

File: README.md [hunk 3]
```diff
@@ -507,9 +509,9 @@ Content-Type: application/json
 [
     {
         "id": 1,
-        "name": "StandardLibrary - localadmin",
+        "name": "Jane's fabric elements",
         "type": "elements",
-        "items": {
+        "data": {
```
Comment (paulfurley) [inline README.md]: why the change from `items` to `data` ? which is correct?

Comment (idrysdale) [reply README.md]: `data` is correct. I only noticed when looking at the response!

**Comment by idrysdale** [comment]
Thanks for the comments, I was too hasty with that! I've accepted them, but fixed them up into the first commit and force pushed :)
